README.md

```diff
@@ -124,6 +125,64 @@ The actual resource consumption may be higher due to additional resources requir
+First information about the failure can be found in the "Logging" tab.
+The most common indicator you'll see there is a log message similar to "request timed out".
+
+Downloading Terasology via the launcher can fail for multiple reasons including:
+1. GitHub being unavailable
+2. the Terasology artifact being unavailable
+3. your internet connection being slow
+4. something on your computer or in your network slowing down the download
+
+To check whether GitHub is available, please refer to https://www.githubstatus.com/.
+
+To check whether the Terasology artifact is available, please go to our [Terasology release page](https://github.com/MovingBlocks/Terasology/releases) and look for the version you were trying to download.
+You should find it at https://github.com/MovingBlocks/Terasology/releases/tag/v<version>.
+At the bottom of the release information below the changelogs, there should be an "Asset" section.
+If this section includes a `TerasologyOmega.zip` file, the Terasology artifact is available.
+If not, please open up a [GitHub Issue](https://github.com/MovingBlocks/TerasologyLauncher/issues/new) or inform us about this on [Discord](https://discord.gg/terasology).
+
+If the Terasology artifact and GitHub are available, but your download timed out, this is likely due to a slow internet connection or something on your computer or in your network slowing down the download.
+In this case, you can either try to hunt down the issue yourself or try to work around the issue manually
+
+### Manual Download Timeout Workaround
+
+You can work around the launcher's timeout logic by downloading the Terasology artifact manually.
+If you have Java installed and set up, you can start Terasology directly from the artifact (scenario 1 below).
+Otherwise, you'll need to copy the artifact into the launcher directory (scenario 2 below).
+If scenario 1 should not work, you might have an incompatible version or flavor of Java installed or an incompatible setup.
+In this case, please go for scenario 2.
+
+Workaround steps:
+1. Download the desired Terasology artifact from GitHub: https://github.com/MovingBlocks/Terasology/releases/download/v<version>/TerasologyOmega.zip
+2. Extract the artifact (zip archive)
+
+Scenario 1: You have Java installed and set up
+3. Start Terasology<br>
+   On Windows: Execute `Terasology.bat`<br>
+   On Linux: Execute `Terasology`
+
+Scenario 2: You don't have Java installed or set up
+3. Create a new path in the `games` sub-directory of the TerasologyLauncher download directory<br>
+   Path for stable releases: `<games-subdir>/OMEGA/STABLE/<version>`<br>
+   Path for pre-releases: `<games-subdir>/OMEGA/NIGHTLY/<version>`
+4. Copy the contents of the previously downloaded and extracted Terasology artifact into the newly created version directory at the end of the path
+5. Start the launcher and select the desired version, the play icon (:arrow_forward:) should now be available for you to start Terasology
```

Comment:
Not sure what is easier to follow, but I think the "drop into cache" workaround jumps less steps of the launcher (so it would still work even if we do something with the ZIP itself beyond extracting)...